test/workbox-cli/node/lib/logger.js

```diff
@@ -0,0 +1,40 @@
+const expect = require('chai').expect;
+const sinon = require('sinon');
+
+const logger = require('../../../../packages/workbox-cli/src/lib/logger');
+
+describe(`[workbox-cli] lib/copy-workbox-sw.js`, function() {
+  const sandbox = sinon.sandbox.create();
+  // Instead of using before/after hooks, the sandbox is explicitly restored
```

Comment:
I would still add beforeEach and after calls to restore just in case of errors, plug you can stub out each console function and then do `console.log.calledOnce` etc.